packages/components/data-resource/data-resource-create-form.tsx

```diff
@@ -69,7 +69,9 @@ const PROVIDERS = getProviders(StoreType.NS);
 const externalProviders = getExternalProviders(StoreType.NS);
 
 const providerDropdownOptions: (t: TFunction) => JSX.Element[] = (t) =>
-  _.map(PROVIDERS, (v, _) => <SelectOption key={v} value={v} />);
+  _.map(PROVIDERS, (v, _) => (
+    <SelectOption key={v} value={v} data-test-dropdown-menu={v} />
```

Comment:
Is it expected that key and the value have the same value? Wouldn't be better that the key receives the proper key?